x-pack/plugins/security_solution/public/cases/pages/index.tsx

```diff
@@ -147,7 +148,7 @@ const CaseContainerComponent: React.FC = () => {
             },
           },
           useFetchAlertData,
-          userCanCrud: userPermissions?.crud ?? false,
+          permissions: casesPermissions,
```

Comment:
Hi Jon, I see we still need to pass the `permissions` here, even though we already have the `permissions` passed to the `CasesContext` at the app level.
I checked the `getCases` and I see the reason is that this function also adds a context, so it also needs the `permissions` (and the `owner`) to be passed here, so my question is: 
Why does the `getCases` need to add another nested context? Since we already added the explicit context at the top level, shouldn't we be able to remove the context in the `getCases` function?

Reply:
As we discussed offline, as this is out of the scope of Jonathan's PR, I open the following issue https://github.com/elastic/kibana/issues/135653 to move the nested provider from the exported components.